python/revng/profile/simple_daemon.py

```diff
@@ -0,0 +1,72 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import atexit
+import os
+import shutil
+import subprocess
+from signal import SIGINT, SIGKILL
+from time import sleep
+
+import requests
+
+from revng.scripting import DaemonProject
+
+from .base import Profile
+
+
+class SimpleDaemon(Profile):
+    def __init__(self, ip: str = "127.0.0.1", port: int = 8000, connection_retries: int = 10):
+        self.ip = ip
+        self.port = port
+        self.connection_retries = connection_retries
+        self.revng_executable = ""
+
+    def start_daemon(self, revng_executable: str = "", resume_dir: str = "") -> None:
```

Comment:
`revng_executable` should be a parameter of the profile, the assertion should happen in `__init__`
`resume_dir` should be a parameter of `get_project`